cg/cli/add.py

```diff
@@ -27,7 +27,6 @@
 @click.group()
 def add():
     """Add new things to the database."""
-    pass
```

Comment:
I don't think we should remove the `pass`es

Reply:
Good catch, seems like the tool automatically removed these since they are not needed. Maybe we want to be explicit anyway. I'm restoring them.